.vscode/settings.json

```diff
@@ -34,6 +34,9 @@
 		"editor.defaultFormatter": "biomejs.biome"
 	},
 	"editor.defaultFormatter": "biomejs.biome",
+	"editor.codeActionsOnSave": {
```

Comment:
@iyansr can you deocuple lint related into another PR? 
we can merge the feature first and lint later